src/fastcs/transports/graphql/graphql.py

```
@@ -17,18 +17,42 @@
+    def __init__(self, controller_apis: list[ControllerAPI]):
+        self._controller_apis = controller_apis
         self._app = self._create_app()
 
     def _create_app(self) -> GraphQL:
-        api = GraphQLAPI(self._controller_api)
-        schema = api.create_schema()
-        app = GraphQL(schema)
+        queries: list[StrawberryField] = []
+        mutations: list[StrawberryField] = []
+        for controller_api in self._controller_apis:
+            id = controller_api.path[0]
```

Ruff (0.15.13)

[error] Variable `id` is shadowing a Python builtin

(A001)